calico-policies/private-network-isolation/eu-es/allow-private-services.yaml

```diff
@@ -28,19 +28,27 @@ spec:
   - action: Allow
     destination:
       nets:
-      # IBM Cloud Monitoring with Sysdig: eu-de
-      # Taken from: https://cloud.ibm.com/docs/monitoring?topic=monitoring-endpoints
-      - 166.9.28.71/32
```

Comment:
Why are you removing these three from this policy, but not from the policy above?  I think these two policies should be consistent.  Also, you should only remove IPs from these policies if you know that no clusters (even older clusters) are using these old IPs.

Reply:
Ideally these files should only contain the IPs relevant for the region. These `eu-de` IPs don't belong in `eu-es`, though these may have been added given Monitoring was a second-wave service in Madrid, so users likely pointed to `eu-de` in the interim.

I'll leave the originals in here for now in case there are still services in this state